.github/workflows/stubtest_stdlib.yml

```diff
@@ -1,43 +0,0 @@
-name: Run stubtest on the stdlib
-
-on:
-  workflow_dispatch:
-  push:
-    branches:
-      - main
-      - master
-    pull_request:
```

Comment:
Is this overindented? In tests.yml it's at the same level as `push:`.

Reply:
Good spot! I believe this is the reason.

Reply:
Oh wow. 🤦 Thanks!

Reply:
Yup, that was it!